src/engine/Engine.ts

```diff
@@ -401,6 +408,9 @@ O|===|* >________________>\n\
          this.displayMode = DisplayMode.FullScreen;
       }
 
+      if (options.backgroundColor) {
+         this.backgroundColor = options.backgroundColor;
```

Comment:
Let's do a `options.backgroundColor.clone()` to ensure we pass a clone of the instance to avoid issues later.